Makefile

```
@@ -56,5 +56,8 @@ fix-image-links:
 fix-image-links-apply:
 	python3 scripts/fix_markdown_image_links.py --apply
 
+# Run all checks
+test: build check-images
+
 # Default target
-all: install serve 
\ No newline at end of file
+all: install serve
```

checkmake (0.2.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)